train.py

```diff
@@ -47,6 +47,11 @@
                     help='quickly test the setup by running over only a small number of events')
 parser.add_argument('--lr-finder', type=str, default=None,
                     help='run learning rate finder instead of the actual training; format: ``start_lr, end_lr, num_iters``')
+parser.add_argument('--tensorboard', type=str, default=None,
+                    help='create a tensorboard summary writer with the given comment')
```

Comment:
... given comment -> given name/(directory?)

Reply:
Ah, I see it's indeed a `comment` :)

Reply:
haha I am just making the modification. (Both are fine I think, though the "name" could be better for understanding)
Thanks for merging :)